civihr_employee_portal/views/views_export/views_my_details_block.inc

```diff
@@ -201,11 +201,11 @@ $handler->display->display_options['fields']['home_email']['table'] = 'hrjc_cont
 $handler->display->display_options['fields']['home_email']['field'] = 'home_email';
 $handler->display->display_options['fields']['home_email']['relationship'] = 'hrjc_contact_details';
 $handler->display->display_options['fields']['home_email']['label'] = 'Personal E-mail';
-/* Field: CiviCRM Contacts: Aggregated Contact Address */
-$handler->display->display_options['fields']['aggregated_address']['id'] = 'aggregated_address';
-$handler->display->display_options['fields']['aggregated_address']['table'] = 'civicrm_contact';
-$handler->display->display_options['fields']['aggregated_address']['field'] = 'aggregated_address';
-$handler->display->display_options['fields']['aggregated_address']['label'] = 'Address';
+/* Field: CiviCRM Contacts: Aggregated Contact Home Address */
+$handler->display->display_options['fields']['aggregated_personal_address']['id'] = 'aggregated_personal_address';
```

Comment:
Should this rather be aggregated_home_address ? Its a bit conflicting since the label says Home Address.
Also the handler.